client/src/pages/UserLogin/UserLogin.js

```diff
@@ -1,29 +1,33 @@
-import React, { useState } from 'react';
-import LogInContainer from './components/LogInContainer';
-import axios from 'axios';
+import React, { useContext, useState } from "react";
+import LogInContainer from "./components/LogInContainer";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { StateContext } from "../../App";
 
 const UserLogin = () => {
   const [logIn, setLogIn] = useState(true);
+  const { setState } = useContext(StateContext);
+
+  const navigate = useNavigate();
 
   const handleChange = () => setLogIn(!logIn);
 
   const handleLogIn = async (username, password) => {
     const response = await axios.post(
-      'http://localhost:5000/api/v1/user/login',
+      "http://localhost:5000/api/v1/user/login",
       {
         username: username,
         password: password,
       }
     );
-    console.log(response.data);
-    //ari mo store sa jwt token sa local storage
-    //nya redirect to home page
+    setState(response.data);
```

Comment:
temporary paman ni sa pero ang correct nga way ani pag set sa state is,

`setState({token: response.data.token})`

Reply:
even better kay i butang ug iyang own nga constant ang data

`const data = response.data`
`setState({token: data.token})`